python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -197,6 +197,17 @@ def test_range(self):
             .equals(self.spark.range(start=0, end=10, step=3, numPartitions=2).toPandas())
         )
 
+    def test_empty_dataset(self):
+        self.assertTrue(
```

Comment:
Let's add a comment with a JIIRA like `# SPARK-XXXX: ...` (https://spark.apache.org/contributing.html)

Reply:
+1. Maintaining contribution style in this module.